api.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios'; 
+const ncNews = axios.create({
+    baseURL: 'https://nc-news-api-qqkg.onrender.com'
```

Comment:
Great to see you have a base URL set up here that can be used for all your request functions 👍